tests/jsmarionette/client/marionette-client/lib/marionette/client.js

```diff
@@ -490,34 +470,37 @@
         args[0] = err;
       }
 
-      return callback.apply(this, args);
+      return cb.apply(this, args);
     },
 
     /**
      * Sends request and formats response.
      *
-     *
      * @private
      * @method _sendCommand
      * @chainable
-     * @param {Object} command marionette command.
-     * @param {String} responseKey the part of the response to pass \
-     *                             unto the callback.
-     * @param {Object} callback wrapped callback.
+     * @param {Object} body The body of the Marionette command
+     * @param {Object} cb wrapped callback
+     * @param {String} key optional key in the response to pass
+     *     unto the callback, will return the full object if undefined
      */
-    _sendCommand: function(command, responseKey, callback) {
-      var self = this;
-
+    _sendCommand: function(body, cb, key) {
```

Comment:
Ditto to gareth's comment. It's standard with node in general and node libraries et al that the callback is last. Unless there's a good reason to change that, I think we'd prefer to keep the arg order as is.